jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-view.js

```diff
@@ -69,6 +69,14 @@ define([
     }
 
     PolymerSupport.init();
+    
+    // register delegate handlers that persist across dashboard mode changes  
+    $('#notebook_panel').on('change', '.show-code-label input', function() {
+        $('#notebook-container').toggleClass('hide-code', !this.checked);
+    });
+    $('#notebook_panel').on('click', '.more-info-link', function() {
+        $(this).parents('.help-area').toggleClass('show-more', 200);
+    });
```

Comment:
What about this:

1. Keep event handlers in template. Locality of code is good and means we won't have to search through code if we change structure of template in the future.
2. Use event namespaces: i.e. `.on('change.DashboardHelpArea')`
3. Before calling `@helpArea.remove()`, call `.off('.DashboardHelpArea')` to remove all event handlers